axelor-contract/src/main/java/com/axelor/apps/contract/service/ContractService.java

```diff
@@ -61,7 +67,7 @@
    * @param date
    */
   @Transactional(rollbackOn = {AxelorException.class, Exception.class})
```

Comment:
No need to specify rollbackOn here
https://github.com/google/guice/wiki/Transactions